eth-rpc/clientdispatch.nim

```diff
@@ -1,43 +1,50 @@
-import asyncnet, asyncdispatch, tables, json, macros
```

Comment:
Since the client and server modules are part of the API, why don't we call them just `json_rpc/client` and `json_rpc/server`?